src/kakarot/precompiles/precompiles.cairo

```diff
@@ -47,28 +47,31 @@ namespace Precompiles {
         calldata: felt*,
         value: felt,
         calling_context: model.ExecutionContext*,
-        return_data_len: felt,
-        return_data: felt*,
+        return_info: model.ReturnInfo*,
     ) -> model.ExecutionContext* {
         alloc_locals;
 
         // Execute the precompile at a given address
         let (output_len, output, gas_used) = _exec_precompile(address, calldata_len, calldata);
 
         // Copy results of precompile to return data
-        memcpy(return_data, output, output_len);
+
+        memcpy(return_info.data, output, output_len);
```

Comment:
here how do we know return_info.data is empty?
meaning that if `return_info.data` already has some values, this would fail?